README.md

```diff
@@ -247,3 +247,25 @@ const container = await new GenericContainer("redis")
   .withWaitStrategy(Wait.forHealthCheck())
   .start();
 ```
+
+## Log Config
```

Comment:
This should probably be further up with the other examples, it's currently in the wait strategies section

Reply:
You're right, I didn't quite get the structure I have to admit. Moved it further up to a point where I think it makes sense.